### PR TITLE
tinyfecvpn: init

### PR DESCRIPTION
**Pull request by ghost**
###### Motivation for this change

I packaged this a while ago and found it while cleaning up my home directory. Thought it might be useful for others.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

###### Notify maintainers

cc @

### CODE REVIEW

**Comment by petabyteboy** [comment]
@GrahamcOfBorg build

**Comment by petabyteboy** [comment]
@GrahamcOfBorg build tinyfecvpn

File: pkgs/tools/networking/tinyfecvpn/default.nix [hunk 1]
```diff
@@ -0,0 +1,33 @@
+{ stdenv, fetchFromGitHub, pkgconfig }:
+
+stdenv.mkDerivation rec {
+  pname = "tinyfecvpn";
+  version = "master";
```
Comment (Lassulus) [inline pkgs/tools/networking/tinyfecvpn/default.nix]: either the version should be "unstable-2019-05-02" or we should use a released version like "20180820.0"

Comment (petabyteboy) [reply pkgs/tools/networking/tinyfecvpn/default.nix]: I changed it to the stable version

**Review by Lassulus — APPROVED** [review]
tested with nix-review, seems to work correctly

**Pull request merged**